src/components/Header.jsx

```diff
@@ -0,0 +1,47 @@
+import { Link, NavLink } from "react-router-dom"
+import { useSelector } from 'react-redux'
+
+import '../styles/header.scss'
+
+const Header = ({ searchMovies }) => {
+  
+  const { starredMovies } = useSelector((state) => state.starred)
+
+  return (
+    <header>
+      <Link to="/" data-testid="home" onClick={() => searchMovies('')}>
+        <i className="bi bi-film" />
+      </Link>
+
+      <nav>
+        <NavLink to="/starred" data-testid="nav-starred" className="nav-starred">
+          {starredMovies.length > 0 ? (
+            <>
+            <i className="bi bi-star-fill bi-star-fill-white" />
+            <sup className="star-number">{starredMovies.length}</sup>
```

Comment:
please use styling to define the styles instead of stylized tags for UI interface design